tests/unit/amo/pages/TestGuides.js

```diff
@@ -56,19 +45,21 @@ describe(__filename, () => {
     );
   };
 
-  it('fetches guides addons', () => {
+  it('fetches the add-ons for a guide page', () => {
```

Comment:
I am not sure because that's not possible.

Reply:
It's not currently possible, but what if there was a link from one guide page to another? Then it would happen, wouldn't it? Or does the router prevent that?

Reply:
If that'd happen, then one would write a test case for it :)